session/session_test.go

```diff
@@ -3,18 +3,21 @@ package session
 import (
 	"testing"
 	"time"
-
+	zookeeper "github.com/Shopify/gozk"
 	toxiproxy "github.com/Shopify/toxiproxy/client"
 )
 
-func TestReceiveEventWhenSubscribing(t *testing.T) {
+func CreateProxy(t *testing.T) (*toxiproxy.Proxy){
 	client := toxiproxy.NewClient("http://localhost:8474")
-	proxy := client.NewProxy(&toxiproxy.Proxy{Name: "gozk_test_zookeeper", Listen: "localhost:27445", Upstream: "localhost:2181", Enabled: true})
-
-	err := proxy.Create()
+	proxy, err := client.CreateProxy("gozk_test_zookeeper", "localhost:27445", "localhost:2181")
 	if err != nil {
 		t.Fatal("Couldn't create proxy. Is toxiproxy running? Error: ", err)
 	}
+	return proxy
+}
+
+func TestReceiveEventWhenSubscribing(t *testing.T) {
```

Comment:
It was initially broken. It was hanging after the disconnect and never being reconnected. 

Once that was fixed it was flaky at 100 miliseconds which is why I made the change to 5 seconds. I could lower it but I don't think it speeds up the tests by much. It definitely needs more than 1s as it more than likely fails then. 

with 2s
```
vagrant@vagrant:~/src/go/src/github.com/Shopify/gozk-recipes/session$ go test
PASS
ok  	github.com/Shopify/gozk-recipes/session	3.197s
```

with 5s
```
vagrant@vagrant:~/src/go/src/github.com/Shopify/gozk-recipes/session$ go test
PASS
ok  	github.com/Shopify/gozk-recipes/session	3.210s
```